src/components/PageHeader/index.js

```diff
@@ -3,15 +3,15 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import SectionHeader from 'components/SectionHeader';
 
-const PageHeader = ({ description, children, contentType }) => (
+const PageHeader = ({ description, children, contentType, columnWidth }) => (
   <div
     className={
       'coa-PageHeader ' + (contentType ? `coa-PageHeader--${contentType}` : '')
     }
   >
     <div className="wrapper container-fluid">
       <div className="row">
-        <div className="col-xs-12 col-md-8">
+        <div className={ "col-xs-12 col-md-"+(columnWidth ? columnWidth : "8") }>
```

Comment:
Allowing the column width to be customizable for Service and Information pages that are with OR without the PageIsAPartOfContainer component.